.github/workflows/mobile-release.yml

```
@@ -15,7 +20,6 @@ jobs:
               || !contains(github.actor, '[bot]'))
       }}
     name: android_release_artifacts
-    uses: ./.github/workflows/_env.yml
     runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-22.04
    needs:
    - env
```

for the  output of the `env` job to be used (ie `needs.env.outputs...`) in a subsequent job, it needs `needs`

---

thanks @phlax !  done!